README.md

```diff
@@ -35,12 +35,10 @@ pip install xdsl
 
 ## Using xDSL
 
```

Comment:
We could add some text here...

XDSL...can be useful for A and B....The following tutos....can help to...educate....etc

Reply:
That's a great idea. @PapyChacal, do you want to add such a sentence?

Reply:
I feel the first paragraph is somehow restarting what was already at the top of the README? Maybe just use the 2nd paragraph only?

Reply:
Oh I missed your comment while writing mine, I agree, I'll push without the first paragraph and see.

Reply:
Nice. That looks good new from my perspective. I wonder what @georgebisbas thinks